app/controllers/two_factor_authentication/otp_verification_controller.rb

```diff
@@ -20,6 +20,9 @@ def show
     end
 
     def create
+      if UserSessionContext.confirmation_context?(context)
+        increment_mfa_selection_attempt_count(:otp)
```

Comment:
@aduth should this (and others relative to their constants) be
```suggestion
        increment_mfa_selection_attempt_count(PIV_CAC)
```
for example regarding https://github.com/18F/identity-idp/pull/11293#discussion_r1812888083

Reply:
Got that in there. It also made the change here https://github.com/18F/identity-idp/pull/11293#discussion_r1812694878 no longer needed.

Reply:
I still see the symbol `:otp` used here. Would we want to switch between `TwoFactorAuthenticatable::AuthMethod::SMS` or `TwoFactorAuthenticatable::AuthMethod::VOICE` like we're doing with WebAuthn, to be able to better differentiate what delivery method they're failing with?